module/gurps.js

```diff
@@ -2514,16 +2517,6 @@ if (!globalThis.GURPS) {
       await handleCombatTurn(combat, turn)
     })
 
-    Hooks.on('deleteCombat', async combat => {
```

Comment:
I'm happy to see more and more low level stuff being pulled out of the catch-all `gurps.js` file and moved to a submodule. Thanks.